### PR TITLE
9.0 mig portal welcome email template

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

File: portal_welcome_email_template/README.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Variables list:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/205/8.0
+   :target: https://runbot.odoo-community.org/runbot/205/9.0
 
 Bug Tracker
```
Comment (pedrobaeza) [inline portal_welcome_email_template/README.rst:61]: Update this to latest template

File: portal_welcome_email_template/__openerp__.py [hunk 2]
```diff
@@ -7,18 +7,18 @@
     "name": "Portal Welcome Email Template",
     "summary": "Adds an customizable email template for portal user "
                "invitation",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "category": "Tools",
     "website": "https://odoo-community.org/",
     'author': 'Incaser Informatica S.L., '
```
Comment (pedrobaeza) [inline portal_welcome_email_template/__openerp__.py]: Please switch this author to our new company `Tecnativa`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little changes

**Review by pedrobaeza — APPROVED** [review]

**Comment by jjscarafia** [comment]
Email template already there on v9 https://github.com/odoo/odoo/blob/9.0/addons/portal/portal_data.xml
:-1:

**Comment by pedrobaeza** [comment]
Ouch, I didn't explore the improvements in that area, so indeed it seems that there's no need for this module. @nicomacr, can you confirm?

**Comment by nicomacr** [comment]
@pedrobaeza yes it's not necessary this module

**Comment by pedrobaeza** [comment]
Then this PR should be converted to the one that removes the module.

**Pull request closed without merging**